src/cache/precomputed_quantities.jl

```diff
@@ -426,6 +444,34 @@ NVTX.@annotate function set_precomputed_quantities!(Y, p, t)
     end
     @. ᶜts = ts_gs(thermo_args..., ᶜspecific, ᶜK, ᶜΦ, Y.c.ρ)
     @. ᶜp = TD.air_pressure(thermo_params, ᶜts)
+    @. ᶜh_tot = TD.total_specific_enthalpy(thermo_params, ᶜts, ᶜspecific.e_tot)
+
+    if turbconv_model isa DiagnosticEDMFX
+        (; ᶜtke⁰) = p.precomputed
+        @. ᶜtke⁰ = Y.c.sgs⁰.ρatke / Y.c.ρ
+    elseif turbconv_model isa PrognosticEDMFX
+        set_prognostic_edmf_precomputed_quantities_draft!(Y, p, ᶠuₕ³, t)
+        set_prognostic_edmf_precomputed_quantities_environment!(Y, p, ᶠuₕ³, t)
+    end
+end
+
+"""
+    set_explicit_precomputed_quantities!(Y, p, t)
+
+Updates the precomputed quantities that are handled explicitly based on the
+current state `Y`.
```

Comment:
Can we also include which functions this impacts? For example: `remaining_tendencies!`, ....

Reply:
The quantities updated in `set_explicit_precomputed_quantities!` are required to compute both `implicit_tendency!` and `remaining_tendency!`; they are just *handled* explicitly. This means that they are updated once per Newton solve, rather than once per Newton iteration. These quantities include things like diffusivity and surface conditions, which affect the implicit tendency when `diff_mode isa Implicit`.

Reply:
Yeah, I'm just saying can we update the doc string to explicitly say this